src/js/api.js

```diff
@@ -0,0 +1,80 @@
+/**         Api class handles all the communication with the API          */
+
+class Api {
+  constructor() {
+    this.baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+    this.newGame = 'Maniac Mansion';
+    this.gameId = null;
+    this.scores = [];
+    this.init(); // Calls the API when the page loads for first time and gets scores
+  }
+
+  /**         Checks if a game id is stored in local storage               */
```

Comment:
[OPTIONAL]
- Kindly remove commented lines.

Reply:
Thanks for the tip. However, I have to ask you. Is not commenting important? All the comments that I left in my code are directed to help the people who I send to my repo take a look when they have doubts. I realize that I was spending a lot of time helping different students, often with the same issue, so I thought that leave some documentation comments would comply with best practices at the same time it frees me from repeating the same explanation once and again.
Please let me know if there is a better way or if I should do it differently.
Thanks for your time!